### DISCUSSION TITLE
Currency Calculator

### DISCUSSION THREAD

**Discussion by StephenHillenbrand** [MarkW06071969/Tafe202206#10] [General]
Hi Team, I have committed a new branch for the currency convertor feature. I have done some testing and it seems to be working ok, the code in the background is rough and could be more efficient but it works.

I would like to do a merge later today with the development branch if you are both ok with that @000961892 @MarkW06071969

**Comment by StephenHillenbrand** [comment]
Can someone please advise the background colour we will be using for the panels so to keep the look and feel the same, 
Should it be the same as the main menu?

**Comment by StephenHillenbrand** [comment]
Currency Calculator has now been completed and merged into the development branch.

There were no merge errors to fix which was great.

**Comment by MarkW06071969** [comment]
Fantastic!, I am still working on the Mortgage Calculator and hopefully have it cloned to GitHub tonight and merge it with the development branch sometime tomorrow (hopefully!)

Sent from Yahoo Mail on Android 
 
  On Wed, 31 Aug 2022 at 11:34 am, ***@***.***> wrote:   


Can someone please advise the background colour we will be using for the panels so to keep the look and feel the same,
Should it be the same as the main menu?
